builder/dockerfiles/Dockerfile

```diff
@@ -32,7 +32,7 @@ RUN cd /tmp \
     && curl -O https://bootstrap.pypa.io/pip/3.6/get-pip.py \
     && python3 get-pip.py
 
-RUN python -m pip install --no-cache-dir torchserve
+RUN python -m pip install --no-cache-dir torchserve nvgpu
```

Comment:
Is it necessary to change to install nvgpu manually here? I think the issue should only happen when the model deployer is using a GPU (https://github.com/facebookresearch/dynabench/blob/d6e8706a04ce54bf92a021ffec512552f0bc9608/builder/utils/deployer.py#L237) in which case it uses the gpu.Dockerfile